src/index.ts

```diff
@@ -377,6 +384,10 @@ class ServerlessAppsyncPlugin {
   async gatherData() {
     const apiId = await this.getApiId();
 
+    if (typeof apiId !== 'string') {
```

Comment:
```suggestion
    if (!apiId) {
```
https://github.com/sid88in/serverless-appsync-plugin/pull/597/files#r1223292215
> if for some reason, `apiId` is an empty string, it probably does not make sense to continue either.